### PR TITLE
Addition of OCI Stage Pipeline

### PR DESCRIPTION
**Pull request by nasx**

### CODE REVIEW

File: ansible/roles/oci-stage-content/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+          app.kubernetes.io/component: "{{ quay_image_path_basename }}-{{ quay_image_tag }}-httpd"
+          app.kubernetes.io/instance: "{{ quay_image_path_basename }}-{{ quay_image_tag }}-httpd"
+        name: "{{ quay_image_path_basename }}-{{ quay_image_tag }}-httpd"
+        namespace: rfe
+      spec:
+        ports:
+        - name: 80-tcp
+          port: 80
+          protocol: TCP
+          targetPort: 80
```
Comment (nasx) [inline ansible/roles/oci-stage-content/tasks/main.yaml:126]: @sabre1041 should we add 443 as well?

Comment (sabre1041) [reply ansible/roles/oci-stage-content/tasks/main.yaml:126]: SSL isnt setup in those images so no

File: ansible/roles/oci-stage-content/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        selector:
+          deployment: "{{ quay_image_path_basename }}-{{ quay_image_tag }}-httpd"
+        sessionAffinity: None
+        type: ClusterIP
+    state: present
+    validate_certs: yes
+
+- name: "Create Route for {{ quay_image_path_basename }}:{{ quay_image_tag }}"
```
Comment (sabre1041) [inline ansible/roles/oci-stage-content/tasks/main.yaml:134]: Any interest in allowing for the `host` field to be set as a pipeline input?

Comment (nasx) [reply ansible/roles/oci-stage-content/tasks/main.yaml:134]: I'm not opposed. Maybe convert the embedded Route resource into a Jinja template and omit the host field if the pipeline input is not provided? Something like this:
```
{% if host_pipeline_input is defined %}
    host: {{ host_pipeline_input }}
{% endif %}
```

Comment (sabre1041) [reply ansible/roles/oci-stage-content/tasks/main.yaml:134]: a empty string is also valid as it will just auto generate

Comment (nasx) [reply ansible/roles/oci-stage-content/tasks/main.yaml:134]: Done.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**